### PR TITLE
Match WP examples for path building

### PR DESCRIPTION
**Pull request by joemcgill**
While creating a patch for merging our functions into core @getsource suggested we mimic the way file paths are built at the end of `image_get_intermediate_size()`.

### CODE REVIEW

**Comment by joemcgill** [comment]
Looking into why the tests are breaking.

File: wp-tevko-responsive-images.php [hunk 1]
```diff
@@ -196,13 +196,9 @@ function tevkori_get_srcset_array( $id, $size = 'thumbnail' ) {
...
-		'file'   => $img_meta['file']
+		'file'   => basename( $img_meta['file'] )
 	);
 
-	if ( strrpos( $img_meta['file'], '/' ) !== false ) {
-		$img_sizes['full']['file'] = substr( $img_meta['file'], strrpos( $img_meta['file'], '/' ) + 1 );
-	}
-
 	// Get the image base url.
 	$img_base_url = substr( $img_url, 0, strrpos( $img_url, '/' ) + 1 );
```
Comment (jaspermdegroot) [inline wp-tevko-responsive-images.php]: ```$img_base_url``` can be removed too since we won't be using it anymore

**Comment by jaspermdegroot** [comment]
@joemcgill 

Looks good to merge to me after removing the unused variable.

**Comment by joemcgill** [comment]
Rebased the three commits into a single commit before merging.

**Comment by jaspermdegroot** [comment]
@joemcgill - Didn't you forget to actually merge the commit or was that on purpose? :)

**Comment by joemcgill** [comment]
Was hoping Travis would resolve whatever issue is going on with the WordPress tests. I'll let this ride for a few days to see if those tests clear up before merging, but everything is passing locally.

**Comment by jaspermdegroot** [comment]
@joemcgill 

Tests are working again \o/
I restarted the test and everything is green. I will merge this into dev.

**Comment by jaspermdegroot** [comment]
Landed in https://github.com/ResponsiveImagesCG/wp-tevko-responsive-images/commit/a64195780968cc784a97954d2ab3706a2ec6782d

**Pull request closed without merging**